src/test/test-seccomp.c

```diff
@@ -837,7 +837,7 @@ static void test_load_syscall_filter_set_raw(void) {
                 assert_se(s = hashmap_new(NULL));
 #if defined __NR_poll && __NR_poll >= 0
                 assert_se(hashmap_put(s, UINT32_TO_PTR(__NR_poll + 1), INT_TO_PTR(-1)) >= 0);
-#else
+#elif defined __NR_ppoll
```

Comment:
hmm, so riscv has neiether poll() nor ppoll()? how does it implement the poll()/ppoll() syscalls then?

we always need to check that the syscall values is >= 0 btw, since libseccomp does some weird stuff and defines unavailable syscall numbers to negative values in some versions.